config/plugins/interactive_environments/jupyter/config/allowed_images.yml.sample

```diff
@@ -0,0 +1,14 @@
+# This file lists acceptable images to allow runing.
+#
+# This allows you, the admin, to create multiple flavours
+# for your users to run. E.g. maybe you need a geosciences flavour,
+# you can create the image based on our default image and add the
+# appropriate `apt-get/pip install` statements.
+---
+-
+    image: bgruening/docker-jupyter-notebook:16.01
+    description: |
```

Comment:
Add a short description to display to the user? I don't think showing the image name is meaningful for users.